image-definitions/kubeflow/components/notebook-controller/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Build the manager binary
+#
+# The Docker context is expected to be:
+#
+# ${PATH_TO_KUBEFLOW/KUBEFLOW repo}/components
+#
+# This is necessary because the Jupyter controller now depends on
+# components/common
+ARG GOLANG_VERSION=1.19
```

Comment:
Bumping version to 1.19 fixes 15 Critical CVEs. `golang` is a builder image, but we will leave it at this version just to have better security posture.